core/src/main/scala/io/finch/package.scala

```diff
@@ -35,4 +39,20 @@ package object finch extends Endpoints with Outputs with ValidationRules {
     type Html = Witness.`"text/html"`.T
     type EventStream = Witness.`"text/event-stream"`.T
   }
+
+  object currentTime {
```

Comment:
I think we can move this under `io.finch.internal` (as it's own file `currentTime.scala`) or inline inside `ToService`. Whatever you prefer.